src/components/SearchLink.tsx

```diff
@@ -33,4 +33,4 @@ export const SearchLink: React.FC<Props> = ({
       {children}
     </Link>
   );
-};
+};
```

Comment:
The component `SearchLink` is correctly implemented according to the provided comments and type definitions. It uses the `getSearchWith` utility to update the search parameters while preserving the existing ones. No changes are needed.